dotcom-rendering/src/client/bootCmp.ts

```diff
@@ -108,7 +107,9 @@ export const bootCmp = async (): Promise<void> => {
 
 	await Promise.all([
 		initialiseCmp(),
-		injectPrivacySettingsLink(),
+		import(
+			/* webpackMode: 'eager' */ '../components/PrivacySettingsLink.importable'
+		),
```

Comment:
This ensures that the `PrivacySettingsLink` island is included in the entry chunk